### PR TITLE
Removed "See More" & "Pagination"

### PR DESCRIPTION
**Pull request by Progi1984**

### CODE REVIEW

File: src/components/AllContributors.vue [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    import EnthusiastsComponent from './Enthusiasts.vue'
+    import BeginnersComponent from './Beginners.vue'
+
+    export default{
+        name: 'allContributors',
+        components: {
+            'Experts': ExpertsComponent,
+            'Enthusiasts': EnthusiastsComponent,
+            'Beginners': BeginnersComponent
+        },
```
Comment (PierreRambaud) [inline src/components/AllContributors.vue]:
```suggestion
    import Experts from './Experts.vue'
    import Enthusiasts from './Enthusiasts.vue'
    import Beginners from './Beginners.vue'

    export default {
        name: 'allContributors',
        components: {
            Experts
            Enthusiasts,
            Beginners,
        },
```

File: src/components/AllContributors.vue [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        computed: {
+            experts: function () {
+                return this.contributors.filter((contributor) => {
+                    return contributor.rank > 10 & contributor.contributions > 25;
```
Comment (PierreRambaud) [inline src/components/AllContributors.vue]:
```suggestion
                    return contributor.rank > 10 && contributor.contributions > 25;
```

File: src/components/AllContributors.vue [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+                });
+            },
+            enthusiasts: function () {
+                return this.contributors.filter((contributor) => {
+                    return contributor.rank > 10 &
+                    contributor.contributions <= 25 &
```
Comment (PierreRambaud) [inline src/components/AllContributors.vue]:
```suggestion
                    return contributor.rank > 10 &&
                    contributor.contributions <= 25 &&
```

File: src/components/AllContributors.vue [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+                    contributor.contributions > 5;
+                });
+            },
+            beginners: function () {
+                return this.contributors.filter((contributor) => {
+                    return contributor.rank > 10 & contributor.contributions <= 5;
```
Comment (PierreRambaud) [inline src/components/AllContributors.vue]:
```suggestion
                    return contributor.rank > 10 && contributor.contributions <= 5;
```

File: src/components/AllContributors.vue [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+                });
+            }
+        },
+        props: [
+            'contributors',
+        ]
```
Comment (PierreRambaud) [inline src/components/AllContributors.vue]:
```suggestion
        props: {
            contributors: {
              type: Array,
              default: () => [],
            },
        }
```

Something like that please :)

File: src/components/TopContributors.vue [hunk 6]
```diff
@@ -23,14 +23,14 @@
...
-        <see-more v-bind:contributors="contributors"></see-more>
+        <all-contributors v-bind:contributors="contributors"></all-contributors>
     </div>
 </template>
 
 <script>
     import ArrowComponent from './ui/Arrow.vue'
     import TopAvatarComponent from './ui/TopAvatar.vue'
-    import SeeMoreComponent from './ui/SeeMore.vue'
+    import AllContributorsComponent from './AllContributors.vue';
```
Comment (NeOMakinG) [inline src/components/TopContributors.vue]:
```suggestion
    import AllContributors from './AllContributors.vue';
```

File: src/components/TopContributors.vue [hunk 7]
```diff
@@ -85,7 +85,7 @@
         components:{
             'arrow': ArrowComponent,
             'top-avatar': TopAvatarComponent,
-            'see-more': SeeMoreComponent
+            'all-contributors': AllContributorsComponent
```
Comment (NeOMakinG) [inline src/components/TopContributors.vue]:
```suggestion
            'all-contributors': AllContributors
```

File: src/components/AllContributors.vue [hunk 8]
```diff
@@ -0,0 +1,46 @@
+<template>
+    <div>
+        <div id="all-contributors" class="col-md-12">
+            <Experts v-bind:contributors="experts"></Experts>
+            <Enthusiasts v-bind:contributors="enthusiasts"></Enthusiasts>
+            <Beginners v-bind:contributors="beginners"></Beginners>
```
Comment (PierreRambaud) [inline src/components/AllContributors.vue]:
```suggestion
            <Experts :contributors="experts"></Experts>
            <Enthusiasts :contributors="enthusiasts"></Enthusiasts>
            <Beginners :contributors="beginners"></Beginners>
```

File: src/components/TopContributors.vue (hunk 6 above)
Comment (PierreRambaud) [inline src/components/TopContributors.vue]: Be consistent with `;`

File: src/components/AllContributors.vue (hunk 5 above)
Comment (Progi1984) [reply src/components/AllContributors.vue]: @PierreRambaud npm doesn't like : 
image

Comment (NeOMakinG) [reply src/components/AllContributors.vue]: because you used an array in props, it's an object 

```
props: { ... }
```

Comment (Progi1984) [reply src/components/AllContributors.vue]: Thanks @NeOMakinG

**Comment by PierreRambaud** [comment]
THis project need an eslint configuration :sweat_smile:

**Comment by PierreRambaud** [comment]
Thanks @Progi1984

**Comment by ttoine** [comment]
when will this be in production?

**Pull request merged**